ecoscope_workflows/tasks/analysis/_aggregation.py

```diff
@@ -0,0 +1,25 @@
+from typing import Annotated
```

Comment:
```suggestion
from typing import Annotated, Literal
```
Goes with `func_name` type hint suggestion below.